lib/icedfrisby.js

```diff
@@ -1051,10 +1050,9 @@ class Frisby {
    * })
    */
   afterJSON (cb) {
-    const self = this
-    this.current.after.push(function() {
-      const responseHeaders = _jsonParse(self.current.response.headers)
-      const bodyJSON = _jsonParse(self.current.response.body)
+    this._hooks.after.push(() => {
+      const responseHeaders = _jsonParse(this.current.response.headers)
+      const bodyJSON = _jsonParse(this.current.response.body)
       cb.call(this, bodyJSON, responseHeaders)
```

Comment:
Oops, this is changing the context for this callback ^^

Hmm, I wonder whether that is necessary, or even desirable.

Reply:
I've never used afterJSON, tbh. If there were a common & specific use-case for afterJSON, we'd likely already have it as an expect (although am now thinking I could use it in jsonrefchecks plugin instead of just `after()`)

You could undo the arrow-ing to restore the previous, although this makes it consistent with after() and I agree that it's probably not even a problem. Would you ever need the context as a test developer?

Reply:
I can't think of a good reason to need the context. It's not Mocha's best side, using the context. If you did, for some rare reason, you could do `const mochaContext = this` before invoking Frisby. `this.skip()`? That's a possibility. If a plugin wanted to do that they'd need a way to access the Mocha context via the Frisby object.

Why don't I undo the arrowing here, and then do another PR that eliminates this and all the places we try to preserve the Mocha context. And we can either add an accessor, or not.

Reply:
I've pushed that tiny revert.